packages/static/kairos-overlay-files/files/system/oem/52_installer.yaml

```diff
@@ -31,11 +43,14 @@ stages:
         - echo "tty1::respawn:/usr/bin/kairos-agent interactive-install --shell tty1" >> /etc/inittab
   boot:
     - if: |
-          [ "$(kairos-agent state get boot)" == "livecd_boot" ] && [ -f "/sbin/rc-service" ]
+        [ "$(kairos-agent state get boot)" == "livecd_boot" ] && [ -f "/sbin/rc-service" ]
```

Comment:
should there not be a 

```
"$(kairos-agent state get kairos.init)" == "openrc"
```

here instead?

Reply:
mmhm yeah, good point. Becuase openrc is "simpler" to check I forgot about using the state xD

Reply:
done!